infrastructure/store/fake.js

```diff
@@ -3,7 +3,24 @@ const latency = 5
 class Store {
   constructor() {
     this.products = []
-    this.orders = []
+    this.orders = [
```

Comment:
please make sure here you include all fields present in makeOrder. We need that for it to be consistent with dynamically created orders later